lib/LedgerSMB/Template/LaTeX.pm

```diff
@@ -105,6 +105,7 @@ sub initialize_template {
 
     my %options = ( format => $config->{_format} );
     Template::Plugin::Latex->new($template->context, \%options);
+    $template->context->define_filter('escape', \&escape);
```

Comment:
Rather than using our own LaTeX escape routine, can we use the latex_encode filter thats part of Template::Plugin::Latex?